JS/popupPage.js

```diff
@@ -0,0 +1,123 @@
+import { projectsarray, createProjects } from './projects.js';
+
+// popup page
+// get the div that includes the popup page
+
+const projectPopup = document.querySelector('.popupPage');
+const projectBtn = document.querySelectorAll('.projectBtn');
+
+projectBtn.forEach((btn, btnIndex) => {
+  btn.addEventListener('click', () => {
+    createProjects();
```

Comment:
Consider not executing this function inside her because each time the modal is fired, it creates more cards added to the previous ones and it messes up the UI, consider using a better approach.

- Either executing the modal pop-up in the function used to create the cards so that the events can be attached on page load
- Go directly inside the function that creates the card and attach the events listener there
- if you have a better suggestion, feel free to try as long as it works 😄 

Here is what the UI looks like after firing the modal couple of times:
<img width="758" alt="svs" src="https://github.com/GeekyHacks/My-Personal-Portfolio/assets/84930355/43a521dd-0931-4800-ba04-8eda42c9eb44">